### ISSUE TITLE
PeoplePicker component doesn't read out position of highlighted result in JAWS. 

### ISSUE THREAD

**Issue by lukehumphrey** [OfficeDev/office-ui-fabric-react#7619]
PeoplePicker component doesn't read out position of highlighted result in JAWS/Chrome. 
Logged by WiPro against Project's Roadmap feature. Logging here per Micah.

Repro steps:
Search for a person using the component 
Navigate through results

Result: SR reads out name of person
Expected: SR reads out name of person and X of Y for the position/total results.

**Comment by lukehumphrey** [comment]
Per Micah should go to Jon Schectman (PROGRAMMER STS).

**Comment by khmakoto** [comment]
Hi @lukehumphrey this doesn't reproduce now on our supported scenario of Narrator+Edge. Can you check if you can still reproduce it in Jaws+Chrome and reply back here with your experience? Thanks!
